mixin/thanos/alerts/rule.libsonnet

```
@@ -153,6 +153,33 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'ThanosRuleNoEvaluationForLast10Intervals',
```

This is more of a nitpick or maybe even opinion already: 

```suggestion
            alert: 'ThanosRuleNoEvaluationFor10Intervals',
```